include/SFML/Graphics/Image.hpp

```diff
@@ -155,6 +155,26 @@ class SFML_GRAPHICS_API Image
     ////////////////////////////////////////////////////////////
     bool saveToFile(const std::string& filename) const;
 
+    ////////////////////////////////////////////////////////////
+    /// \brief Save the image to a buffer in memory
+    ///
+    /// The format of the image must be specified.
+    /// The supported image formats are bmp, png, tga and jpg.
+    /// This function fails if the image is empty, or if
+    /// the format was invalid.
+    ///
+    /// \param output Buffer to fill with encoded data
+    /// \param format Encoding format to use
+    /// \param output Buffer to fill with encoded data
+    /// \param format Encoding format to use
```

Comment:
Duplicated `\param` comments

Reply:
Good spot, must have messed it up resolving conflicts - fixed now